bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -68,8 +68,6 @@ get_size_for_available_space = (use_width, use_height, client_width, client_heig
 
 # TODO (bev) PlotView should not be a RendererView
 class PlotView extends Renderer.View
-
-  className: "bk-plot"
```

Comment:
> 'm not certain of the ramifications of this.

There should be none. However, there are multiple `bk-plot-something` classes and that's why I didn't want `bk-plot` used at the top-level (to avoid confusion).